lib/Varien/Io/File.php

```diff
@@ -209,7 +209,8 @@ public function streamReadCsv($delimiter = ',', $enclosure = '"')
         if (!$this->_streamHandler) {
             return false;
         }
-        return @fgetcsv($this->_streamHandler, 0, $delimiter, $enclosure);
+        $escape = '\\';
+        return @fputcsv($this->_streamHandler, $row, $delimiter, $enclosure, $escape);
```

Comment:
PHPStan is right, this was an existing issue before this PR. 

The function ```streamReadCsv($delimiter = ',', $enclosure = '"')``` gets only two parameters comparing with ```streamWriteCsv(array $row, $delimiter = ',', $enclosure = '"')``` which gets $row as parameter.

Reply:
$row should be changed back to 0.